pkg/apis/crd.nsx.vmware.com/v1alpha1/ipblockdata_types.go

```diff
@@ -0,0 +1,53 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type IPBlockVisibility string
+
+const (
+	IPBlockVisibilityExternal   = "External"
```

Comment:
Ditto

Reply:
Reused the value and updated value, thanks!